shell/browser/browser_linux.cc

```diff
@@ -139,7 +151,35 @@ bool Browser::SetBadgeCount(std::optional<int> count) {
   }
 }
 
-void Browser::SetLoginItemSettings(LoginItemSettings settings) {}
+void Browser::SetLoginItemSettings(LoginItemSettings settings) {
+  dbus::Bus::Options options;
+  auto bus = base::MakeRefCounted<dbus::Bus>(options);
+
+  dbus::ObjectProxy* object_proxy = bus->GetObjectProxy(
+      kFreedesktopPortalName, dbus::ObjectPath(kFreedesktopPortalPath));
+  dbus::MethodCall method_call(kFreedesktopPortalBackground,
+                               "RequestBackground");
+
+  dbus::MessageWriter writer(&method_call);
+  writer.AppendString("");
+
+  dbus::MessageWriter options_writer(nullptr);
+  writer.OpenArray("{sv}", &options_writer);
+
+  dbus::MessageWriter autostart_writer(nullptr);
+  options_writer.OpenDictEntry(&autostart_writer);
+  autostart_writer.AppendString("autostart");
+  autostart_writer.AppendVariantOfBool(settings.open_at_login);
```

Comment:
docs/app.md needs to be updated to show that `open_at_login` is supported on Linux :tada:

Reply:
Added!